content/cockpit-150.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cockpit 150
+author: Martin Pitt
+date: 2017-09-08 10:00
+tags: cockpit, linux
+slug: cockpit-150
+category: release
+summary: Cockpit with systemd and GCE 1.7.x improvements
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from versions 149 and 150.
+
+### Add information about unmet conditions for systemd services
+
+Systemd units can have conditions which must be met in order for the service to
+start. Cockpit now shows unmet conditions in the details of a service, so that
+users can see why a service is in active.
+
+![Unmet systemd service conditions](images/services-conditions.png)
+
+### Automatically enable and start newly created timers on the Services page
+
+Cockpit allows the user to create custom systemd timers on the Services page,
+to run commands at particular times or after boot. Previously they had to be
+explicitly enabled and started to become effective; this now happens
+automatically.
+
+### Support Kubernetes basic auth with GCE 1.7.x
+
+Google Compute Engine 1.7.x introduces authentication provider information
```

Comment:
Add support for kubernetes authentication providers when parsing kube configuration files . Some kubernetes deployments such as GCE use kubernetes authentication providers so that they can automatically regenerate their access token when needed. Cockpit now supports kube config files using this format.

Reply:
Updated, thanks! I trimmed the heading a bit to not get too long.